mono/metadata/mono-basic-block.c

```diff
@@ -336,6 +336,8 @@ bb_formation_il_pass (const unsigned char *start, const unsigned char *end, Mono
 	MonoSimpleBasicBlock *branch, *next, *current;
 	const MonoOpcode *opcode;
 
+	// Note: mono_error_init is not called, assume mono_basic_block_split did so
+
```

Comment:
We should always call mono_error_init () for functions with a MonoError argument, thats the convention used by the rest of the runtime code.